present_conjugator.js

```diff
@@ -0,0 +1,58 @@
+//Left breakdown function in here so that the conjugate() function is easier to test.
+const breakdown = (input) => {
+   const hex = '\\u'+(input.charCodeAt(0)+0x10000).toString(16).slice(1);
+   const numHex = hex.substring(2);
+   total = parseInt(numHex, 16);
+
+   let letterArray = [];
+   total = total - 44032;
+   let initialValue = Math.floor(total/588);
+   letterArray.push(initialValue);
+   total = total - (588*initialValue);
+   let medialValue = Math.floor(total/28);
+   letterArray.push(medialValue);
+   let finalValue = total - (28*medialValue);
+   if (finalValue > 0) {
+     letterArray.push(finalValue);
+   }
+   return letterArray;
+ }
+
+
+
+//Left console.logs in here so that testing the conjugation is easy. Return conjugate and remove console.logs when testing is complete.
+function conjugate(word){
+
+let wordLength = word.length;
+let conjugate = '';
+
+if (word[wordLength-2] === '하'){
+  conjugate = word.slice(0, wordLength-2);
+  conjugate = conjugate.concat('해');
+  console.log('shoked', conjugate);
```

Comment:
i left console.logs in here so that the function can be tested more thoroughly for correct korean conjucation